### PR TITLE
Screenshot Update for gist directory

### LINKED ISSUE #23967: MLH Fellow content update (images)
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### This issue is reserved for our MLH Fellow

Directories 
- [ ] https://github.com/github/docs/tree/main/assets/images/settings
- [ ] https://github.com/github/docs/tree/main/assets/images/help/gist

### What part(s) of the article would you like to see updated?

Update the images with our new [image guidelines](https://github.com/github/docs/blob/main/contributing/images-and-versioning.md).

### Additional information

Feel free to do these one PR at a time or several updates on the same PR. Just link this issue and we will keep it open until all images are updated.

### PR DESCRIPTION
**Pull request by OlayinkaAtobiloye**
### Why:

Closes [ISSUE](https://github.com/github/docs/issues/23967)

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by OlayinkaAtobiloye** [comment]
Hi, @cmwilson21! Updated the screenshots for the [Gist](https://github.com/github/docs/tree/main/assets/images/help/gist) directory here. Could you take a look when chanced? Thanks!

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md [hunk 1]
```diff
@@ -80,4 +80,3 @@ Alternatively, you can drag and drop a text file from your desktop directly into
 ![Drop-down menu to select gist visibility]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:80]: I don't think the `gist_name_description` and `gist_test_box` images are necessary. When you navigate to the Gist homepage, there was only 3 fields. Folks should be able to find them by description alone. 

Instead, let's remove both those images and update the procedural steps to clarify which fields we're talking about. I can't leave this comment on the appropriate lines unfortunately 😅 but something like this:

1. In the "Gist description" field, type a description for your gist.
2. In the "Filename including extension" field,  type a file name for your gist, including the file extensions.
3. In the file contents field, type the text of your gist.

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md [hunk 2]
```diff
@@ -12,11 +12,6 @@ versions:
   ghae: '*'
   ghec: '*'
 ---
-## Forking gists
-
-Each gist indicates which forks have activity, making it easy to find interesting changes from others.
-
-![Gist forks](/assets/images/help/gist/gist_forks.png)
 
 ## Cloning gists
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md:17]: Any information in a screenshot should be in text as well. Let's add a sentence before this screenshot, saying something like "To clone a gist, select the **Embed** dropdown menu, then click **Clone via HTTPS** or **Clone via SSH**."

We also need to update the alt text, and fix the outline (which is too thick).

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md [hunk 3]
```diff
@@ -12,11 +12,6 @@ versions:
   ghae: '*'
   ghec: '*'
 ---
-## Forking gists
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md:15]: I think it would be strange to have an article called "Forking and cloning gists" without mentioning forking at all. Can you clarify why you're removing this section?  I agree the screenshot itself is not particularly useful and we could definitely delete, but would suggest keeping the rest of the section.

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md [hunk 4]
```diff
@@ -28,8 +23,6 @@ If you want to make local changes to a gist and push them up to the web, you can
 
 To view a gist's full commit history, click the "Revisions" tab at the top of the gist.
 
-![Gist revisions tab](/assets/images/help/gist/gist_revisions_tab.png)
-
 You will see a full commit history for the gist with diffs.
 
 ![Gist revisions page](/assets/images/help/gist/gist_history.png)
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md:28]: We need to update this alt text.

Also the image looks a bit blurry to me. Is it 144 dpi as specified [here](https://github.com/github/docs-internal/blob/main/contributing/images-and-versioning.md#technical-specs)?

**Review by lecoursen — COMMENTED** [review]
Thanks for jumping in on the accessibility work @OlayinkaAtobiloye! ✨ Let me know if it was be helpful to set up a Zoom to go over anything in my comments. 😊

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (motebfz) [reply content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:77]: 👍

**Comment by OlayinkaAtobiloye** [comment]
> Thanks for jumping in on the accessibility work @OlayinkaAtobiloye! ✨ Let me know if it was be helpful to set up a Zoom to go over anything in my comments. 😊

Hi @lecoursen, thanks for the feedback! I made some changes accordingly. Could you check it out when chanced, please? thank you!

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md [hunk 5]
```diff
@@ -70,14 +70,9 @@ Alternatively, you can drag and drop a text file from your desktop directly into
...
-
-6. Click **Create secret Gist** or **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.
-  ![Button to create gist](/assets/images/help/gist/create-secret-gist-button.png)
+3. In the "Gist description" field, type an optional description for your gist.
+4. In the "Filename including extension" field, type a file name for your gist, including the file extensions.
+5. In the file contents field, type the text of your gist.
+6. Optionally, to create {% ifversion ghae %}an internal{% else %}a public{% endif %} gist, click {% octicon "triangle-down" aria-label="The downwards triangle icon" %}, then click **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.
+![Screenshot of a visibility drop-down menu on GitHub Gist. A little dropdown icon beside a button labelled "Create secret gist" is outlined in dark orange.]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:77]: The border on this one still looks too thin to me. 

The alt text is great though! Just a few tiny suggestions to make it slightly easier to read, and use American spelling. 

```suggestion

   ![Screenshot of the visibility dropdown menu for a new gist. Next to a button labeled "Create secret gist", a dropdown icon is outlined in dark orange.]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
```

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md [hunk 6]
```diff
@@ -70,14 +70,9 @@ Alternatively, you can drag and drop a text file from your desktop directly into
...
-
-4. Type the text of your gist into the gist text box.
-![Gist text box](/assets/images/help/gist/gist_text_box.png)
-
-5. Optionally, to create {% ifversion ghae %}an internal{% else %}a public{% endif %} gist, click {% octicon "triangle-down" aria-label="The downwards triangle icon" %}, then click **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.
-![Drop-down menu to select gist visibility]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
-
-6. Click **Create secret Gist** or **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.
-  ![Button to create gist](/assets/images/help/gist/create-secret-gist-button.png)
+3. In the "Gist description" field, type an optional description for your gist.
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:73]: https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#procedural-steps

```suggestion
3. Optionally, in the "Gist description" field, type a description for your gist.
```

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md [hunk 7]
```diff
@@ -12,24 +12,23 @@ versions:
...
-![Gist forks](/assets/images/help/gist/gist_forks.png)
-
 ## Cloning gists
 
 If you want to make local changes to a gist and push them up to the web, you can clone a gist and make commits the same as you would with any Git repository. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/cloning-a-repository)."
 
-![Gist clone button](/assets/images/help/gist/gist_clone_btn.png)
+To clone a gist, select the Embed dropdown menu, then click Clone via HTTPS or Clone via SSH.
+
+![Screenshot of the Embed dropdown from GitHub Gist. The Embed dropdown is expanded and an option labeled “Clone via HTTPS” is outlined in dark orange.](/assets/images/help/gist/gist_clone_btn.png)
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md:26]:
```suggestion
![Screenshot of the "Embed" dropdown menu in GitHub Gist. The dropdown is expanded, and an option labeled “Clone via HTTPS” is outlined in dark orange.](/assets/images/help/gist/gist_clone_btn.png)
```

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md [hunk 8]
```diff
@@ -12,24 +12,23 @@ versions:
...
 ## Viewing gist commit history
 
 To view a gist's full commit history, click the "Revisions" tab at the top of the gist.
 
-![Gist revisions tab](/assets/images/help/gist/gist_revisions_tab.png)
-
 You will see a full commit history for the gist with diffs.
 
-![Gist revisions page](/assets/images/help/gist/gist_history.png)
+![Screenshot of the GitHub Gist Revisions page. A tab labeled “Revisions” with dark orange underline is outlined in dark orange.](/assets/images/help/gist/gist_history.png)
```
Comment (lecoursen) [inline content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md:34]: I'm not sure the dark orange underline is important to mention, and it just makes the sentence a bit hard to parse with the second dark orange later. 😊 
```suggestion
![Screenshot of the "Revisions" page in GitHub Gist. A tab, labeled “Revisions”, is outlined in dark orange.](/assets/images/help/gist/gist_history.png)
```

**Review by lecoursen — COMMENTED** [review]
This is looking really good @OlayinkaAtobiloye! ✨ There's one image where the border still looks wrong to me, and I left a few suggestions with small tweaks to phrasing, but this is almost ready to go! 🎉

**Comment by OlayinkaAtobiloye** [comment]
> This is looking really good @OlayinkaAtobiloye! ✨ There's one image where the border still looks wrong to me, and I left a few suggestions with small tweaks to phrasing, but this is almost ready to go! 🎉

Hi @lecoursen, thanks for the feedback! Made some changes; please check them out.✨

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md [hunk 9]
```diff
@@ -70,14 +70,10 @@ Alternatively, you can drag and drop a text file from your desktop directly into
...
 
-4. Type the text of your gist into the gist text box.
-![Gist text box](/assets/images/help/gist/gist_text_box.png)
-
-5. Optionally, to create {% ifversion ghae %}an internal{% else %}a public{% endif %} gist, click {% octicon "triangle-down" aria-label="The downwards triangle icon" %}, then click **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.
-![Drop-down menu to select gist visibility]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
-
-6. Click **Create secret Gist** or **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.
-  ![Button to create gist](/assets/images/help/gist/create-secret-gist-button.png)
+   ![Screenshot of the visibility dropdown menu for a new gist. Next to a button labeled "Create secret gist", a dropdown icon is outlined in dark orange.]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
```
[the remark this answers is not in the document]
Comment (OlayinkaAtobiloye) [reply content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:78]: Hi @lecoursen, oh I missed that; thanks for pointing it out! The image has been updated now!

**Review by lecoursen — DISMISSED** [review]

**Comment by lecoursen** [comment]
Sorry to comment _again_ but I just realized the image for GitHub AE says "public gist", and they're called "internal gists" on GHAE. I thought it might be easier to just provide you the screenshot, since I have easy access to GHAE, and you can crop and highlight from there!

image

**Comment by OlayinkaAtobiloye** [comment]
> Sorry to comment _again_ but I just realized the image for GitHub AE says "public gist", and they're called "internal gists" on GHAE. I thought it might be easier to just provide you the screenshot, since I have easy access to GHAE, and you can crop and highlight from there!
> 
> image

Thanks for this, @lecoursen! The image is now updated. Thanks!

**Review by lecoursen — APPROVED** [review]
🎉 Looks great! I'll go ahead and merge. 😎
